pkg/sql/crdb_internal.go

```diff
@@ -660,8 +660,10 @@ CREATE TABLE crdb_internal.node_statement_statistics (
   service_lat_var     FLOAT NOT NULL,
   overhead_lat_avg    FLOAT NOT NULL,
   overhead_lat_var    FLOAT NOT NULL,
-  bytes_read          INT NOT NULL,
-  rows_read           INT NOT NULL,
+  bytes_read_avg      FLOAT NOT NULL,
```

Comment:
This could be somewhat weird in a mixed version state? Does the admin UI take stats from here or via a gRPC endpoint? Either way, I'm not sure it's a problem since we don't distribute virtual table operations, so a node shouldn't run into a problem seeing a float vs an int.

Reply:
Admin UI reads from gRPC. The trouble with this virtual table would be clients who expect a certain set of columns from this table. When they upgrade, they'd see an unpredictable mix depending on the node they connect to. I hope that nobody relies on these, but it's certainly possible that they do.

Reply:
I'm not sure that this is too big of a deal -- we iterate on these internal table columns somewhat frequently.

Reply:
But we normally just *add* columns. That's the policy - we can add columns at will, but removing/changing old ones is kind of a no no because it breaks backward compat!

Reply:
I think keeping the bytes_read and rows_read columns where you just cast the new values into ints for them, add the new _avg columns, and say that we are deprecating the bytes_read and rows_read columns.